rero_ils/modules/acq_orders/extensions.py

```diff
@@ -43,6 +43,20 @@ def pre_load(self, data, loader=None):
         data.pop('total_amount', None)
         data.pop('status', None)
 
+    def pre_delete(self, record, force=False):
```

Comment:
It should be interesting to either change the extension name, either create an extension with your `pre_delete` hook. Solution#1 seems great (AcquisitionOrderExtension ?)